Classes/Controller/AbstractFluxController.php

```diff
@@ -270,7 +270,7 @@ protected function initializeView(ViewInterface $view)
      */
     protected function resolveView()
     {
-        return $this->objectManager->get(is_callable([$this, 'resolveViewObjectName']) ? $this->resolveViewObjectName() : $this->defaultViewObjectName);
+        return $this->objectManager->get($this->defaultViewObjectName);
```

Comment:
This will be the right solution, but not yet - while we still have support for current LTS of TYPO3 we do need to call the method. Possible check could be:

```php
return $this->objectManager->get(!method_exists($this, 'resolveViewObjectName') ? $this->defaultViewObjectName : ($this->resolveViewObjectName() ?: $this->defaultViewObjectName))`
```